backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/modular/interceptor/PP_Order.java

```diff
@@ -97,4 +103,16 @@ public void onBeforeComplete(@NonNull final I_PP_Order order)
 			modularPPOrderService.validateModularOrder(ppOrderId);
 		}
 	}
+
+	@DocValidate(timings = ModelValidator.TIMING_AFTER_COMPLETE)
+	public void afterComplete(@NonNull final I_PP_Order order)
```

Comment:
as discussed, consider 
* either triggering this event of after close
* either triggering this event on PP_Cost_Collector of type receipt

Reply:
went with "either triggering this event on PP_Cost_Collector of type receipt"